py_gtfs_rt_ingestion/tests/test_ingest.py

```diff
@@ -0,0 +1,19 @@
+from ingest import parseArgs, run
```

Comment:
super duper basic testing stub.

what we want to do here is create a suit of tests that will check that our functions are all doing the things we hope they will.

right now `test_argparse` is just testing out that we're parsing a string in the fname arg. in the future, we'll want to put validation stuff in there, and maybe add some kind of argparse action to let us make selections on schemas based on input filenames. all that will get tested in this block, as well as tests against bad inputs.